examples/terraform-integrations/terraform-workflows-templates.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: terraform-argo-workflows-templates
+  namespace: argocd
+  labels:
+    env: dev
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/cnoe-io/backstage-terraform-integrations
+    targetRevision: main
```

Comment:
Do we want to pin to a version?

Reply:
good call. I'd say at a later point in time. Right now this whole work is experimental and for demo purposes.